### ISSUE TITLE
I want to pull Docker Image of RHEL JBoss EAP 7.1 or later from registry.redhat.io

### ISSUE THREAD

**Issue by krajapraveen** [kubernetes/minikube#7416]
Hi 

I want to pull Docker Image of RHEL JBoss EAP 7.1 or later from registry.redhat.io but I am not able to pull it.

When I specified as:  docker login registry.redhat.io
The output is as follows: 

Authenticating with existing credentials...
WARNING! Your password will be stored unencrypted in C:\Users\srikanth.ganji\.docker\config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded.

But when I tried to pull the Docker Image from registry.redhat.io by giving the following command:

docker pull jboss-eap-7/eap72-openshift 

I am getting the below error: 

Using default tag: latest
Error response from daemon: pull access denied for jboss-eap-7/eap72-openshift, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.

Can anyone help me on this .

**Comment by tstromberg** [comment]
Can you provide an exact list of instructions to replicate this error?
 
Please include your `minikube start` command.

**Comment by krajapraveen** [comment]
Hi @tstromberg 

I need to pull the Red Hat Linux's JBoss EAP 7.1 Dockerfile from registry.redhat.io .

When I logged into registry.redhat.io  by providing the credentials, I got the below successful login message.

$ docker login registry.redhat.io
Authenticating with existing credentials...
WARNING! Your password will be stored unencrypted in C:\Users\srikanth.ganji\.docker\config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded.

When I tried to pull the Dockerfile for Red Hat Linux's JBoss EAP 7.1 from registry.redhat.io, I have got the below issue

$ docker pull jboss-eap-7/eap72-openshift
Using default tag: latest
Error response from daemon: pull access denied for jboss-eap-7/eap72-openshift, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

PFA screenshots for your reference


![1](https://user-images.githubusercontent.com/44344982/78468282-ef4c1680-7733-11ea-9f77-1035b6d6ca9a.JPG)

![2](https://user-images.githubusercontent.com/44344982/78468284-f7a45180-7733-11ea-9307-93a635ef80a6.JPG)
